neural_lam/models/ar_model.py

```diff
@@ -167,38 +209,33 @@ def unroll_prediction(self, init_states, forcing_features, true_states):
                 pred_std_list, dim=1
             )  # (B, pred_steps, num_grid_nodes, d_f)
         else:
-            pred_std = self.per_var_std  # (d_f,)
+            pred_std = self.diff_std  # (d_f,)
```

Comment:
There is a reason to keep `per_var_std` (to include variable weights), so this should not be changed to `diff_std`.

Reply:
Yes, I agree. I will revert this and add the feature-weighting too

Reply:
I reintroduced it